.gitignore

```diff
@@ -6,3 +6,8 @@ build/
 runtime/
 logs/
 /bin/
+.classpath
+.naivebayes-model.crc
+.project
+ivy/ivy-2.3.0.jar
```

Comment:
What about ivy/ivy-2.4.0.jar which has been committed in f9981d4? It's supposed to be "installed" by `ant ivy-download`.

Reply:
Thanks @sebastian-nagel I've updated the PR to also prevent ```ivy-2.4.0.jar``` from being added to SCM.

Reply:
+1 Maybe also remove this file to avoid confusions?

Reply:
and please go forward and commit, thanks!